packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -107,48 +178,54 @@ rm -rf $RPM_BUILD_ROOT
 # record whether or not our service was running when docker is upgraded
 ecs_status=$(/sbin/status ecs 2>/dev/null || :)
 if grep -qF "start/" <<< "${ecs_status}"; then
-	/sbin/stop ecs >/dev/null 2>&1 || :
-	if [ "$1" -ge 1 ]; then
-		# write semaphore if this package is still installed
-		touch %{running_semaphore} >/dev/null 2>&1 || :
-	fi
+        /sbin/stop ecs >/dev/null 2>&1 || :
```

Comment:
I understand changing tabs to spaces, but why 8 spaces?

Reply:
There must have been two tabs here, this was from expanding the tabs. I can clean up the entire file including spacing like this to fix up the extra indent.